quartz/components/scripts/search.inline.ts

```diff
@@ -225,8 +225,12 @@ document.addEventListener("nav", async (e: unknown) => {
     const button = document.createElement("button")
     button.classList.add("result-card")
     button.id = slug
-    button.innerHTML = `<h3>${title}</h3>${htmlTags}<p>${content}</p>`
```

Comment:
i wonder if we should make the full button `a` (we already have tabindex set on it)

Reply:
that should work as well.

Reply:
This is with just `a`

https://github.com/jackyzha0/quartz/assets/29749331/f31a6126-e13c-48a3-a38c-71019445b708

versus the current


https://github.com/jackyzha0/quartz/assets/29749331/2fca6e8a-e3f5-4cea-8b83-d4b407a2f4c4


either way works with me. Though just `a` seems pretty clean as well.

Reply:
let's do `a` instead of `button`

Reply:
sg.